readthedocs/subscriptions/constants.py

```diff
@@ -1,4 +1,39 @@
 """Constants for subscriptions."""
+from django.utils.translation import gettext_lazy as _
 
 # Days after the subscription has ended to disable the organization
 DISABLE_AFTER_DAYS = 30
+
+# Values from `value` that represent an unlimited value.
+UNLIMITED_VALUES = [None, -1]
+
+TYPE_CNAME = "cname"
+TYPE_CDN = "cdn"
+TYPE_SSL = "ssl"
+TYPE_SUPPORT = "support"
+
+TYPE_PRIVATE_DOCS = "private_docs"
+TYPE_EMBED_API = "embed_api"
+TYPE_SEARCH_ANALYTICS = "search_analytics"
+TYPE_PAGEVIEW_ANALYTICS = "pageviews_analytics"
+TYPE_CONCURRENT_BUILDS = "concurrent_builds"
+TYPE_SSO = "sso"
+TYPE_CUSTOM_URL = "urls"
+TYPE_AUDIT_LOGS = "audit-logs"
+TYPE_AUDIT_PAGEVIEWS = "audit-pageviews"
```

Comment:
Is there any reason why these are prefixed with `TYPE_` instead of `FEATURE_`? `TYPE_EMBED_API` does not make too much sense to me, but `FEATURE_EMBED_API` does.

In places like this one, it will be more readable: https://github.com/readthedocs/readthedocs.org/pull/8920/files#diff-e7f36d6ba29b1ae0d1c6e12781ba3f54dc4cb434cf07057f73e2f5d57b545ae9R166

Reply:
This is also why I've preferred model constants are defined on the models themselves. The added context and readability of `Feature.TYPE_EMBED_API` or `Feature.EMBED_API` is quite nicer than a list of constants.

Reply:
This is a place where I'd love to use enums instead p: